hw/top_earlgrey/data/ip/chip_alert_handler_testplan.hjson

```diff
@@ -13,7 +13,11 @@
             - Program the alert_test CSR in each block to trigger each alert one by one.
             - Ensure that all alerts are properly connected to the alert handler and cause the
               escalation paths to trigger.
+            - Ensure sival version of this test makes no alterations to OTP (use NMI escalation
+              path)
```

Comment:
Does this check need to be specifically for the sival version?

Reply:
Well no issue if the non si-val version does an OTP change. As this test already exists it's important for whoever works on the testplan that they check this specifically and make changes if needed so I'd prefer to leave this in.

Reply:
Oh, ok. I think I was really unclear in what I wrote. I meant to say something like "Is this check really only relevant for the sival version?" If the check can apply both sides, I'd drop the prefix. If it is a sival -specific note, maybe it makes sense to add an explicit "Note for silicon targets" section to split it out.